forem/forem
# Updated Admin Index View Layout and feature flagged it.

## What type of PR is this? (check all applicable)


## Description
In this PR I simply move boxes around to match the general of the layout for the new Admin Index View. I've also moved the tabs and filters into its own partials. During subsequent PR's the team will be working on this simultaneously, and having this in place will just make it easier to collaborate on.  

<img width="1511" alt="Screenshot 2022-03-21 at 16 41 19" src="https://user-images.githubusercontent.com/2786819/159285272-298b11f0-2ec3-4fdd-82a3-11d0a53cde4f.png">

There are no accessibility changes in this PR. I just moved things around. 

## Related Tickets & Documents



- Related Issue #
- Closes #

## QA Instructions, Screenshots, Recordings

1. Run `FeatureFlag.enable(:admin_index_view)`
2. Ensure that the updated changes as per the screenshot show up.

### UI accessibility concerns?

We'll tackle the accessibility concerns in the future PR's. Fore now this provides a skeleton for the new view. 

## Added/updated tests?

- [x] No, no functionality has been changed

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](gif_link)
